java/kafka/streams/src/main/java/KafkaStreamsExample.java

```diff
@@ -54,4 +54,4 @@ public static void main(String[] args) {
 
         streams.start();
     }
-}
+}
```

Comment:
Maybe you can revert this to make changes in GitHub more easy to trace.

Reply:
Extra empty line was missing